### ISSUE TITLE
Veggie-Loaded Balsamic Pasta

### ISSUE THREAD

**Issue by Stephanymtr** [iteles/blw-baby#16]
![image](https://user-images.githubusercontent.com/91985721/161055902-eb4c9e68-7e9c-494b-bda9-55a2fedcb42f.png)

https://www.veganrunnereats.com/5262/balsamic-pasta-vegan-recipe/

*going to look after A now, will edit this later*

**Comment by Stephanymtr** [comment]
@iteles how do you feel about the balsamic sauce ? If you think we could try this one out, I'm happy to put a shopping list together for this recipe.

**Comment by nelsonic** [comment]
Yesterday during the fruit volcano science experiment, **A** was liking his fingers that had vinegar on them ... 😋 
So there's a good chance that he like vinegar. We should maybe test _Balsamic_ vinegar with him ... 👅 

Maple syrup is _mega_ sweet. I don't think he's ever had it. @iteles can confirm. 🍬

**Comment by Stephanymtr** [comment]
Yes, I agree! I was thinking we could sub maple syrup for apple sauce or even some banana puree (weird, I know but who knows)

**Comment by iteles** [comment]
Ooooooh if he likes this (or any other) sauce, then we have a great way for him to eat veggies!

I'm not sure if balsamic vinegar or mustard are baby safe as they're very acidic on their little tummies and I have no idea what the ingredients are (not being a fan myself 😄) so please research whether they are @Stephanymtr !
Good places to research are https://comidadebebe.pt (if you can't find answers on the site she often answers DMs on IG) and https://solidstarts.com (we should probably add these to a 'good resources' list in this repo too!

Agree with subbing in applesauce for the maple syrup 👌🏻

**Comment by Stephanymtr** [comment]
I found out that you can start adding vinegar to your baby’s diet starting at the age of 6 months old.
https://canbabieseat.com/vinegar/ It can cause indigestion in excess, so they suggest to use it in a diluted form. 
We can introduce mustard seeds into a baby’s diet starting at the age of 8 months. they should be given in small amounts in order for them to get used to the flavor.
https://canbabieseat.com/mustard/

I found recipes on comida de bebe with those ingredients, but I couldn't find a blog or a post specifically about this subject. I have contacted them on IG, so I'll post here what they say when I hear from them.

**Comment by nelsonic** [comment]
Wow, that’s a genius SEO content farm! 😯

They registered an _excellent_ domain name and then put up generic content with a stock photo of the food parents/caretakers will search for and then make money on the Ads. 

![7E863E22-5869-4148-B9E8-6638C5BA546D](https://user-images.githubusercontent.com/194400/163685856-07bd9cb0-970b-4575-b122-504bc89fe5e7.jpeg)

Zero links to peer-reviewed research or other authoritative content sources though, so I certainly wouldn’t trust it. But I can see why some people would. With that level of SEO and partnership with Google/Facebook in terms of Ads https://canbabieseat.com/terms-privacy/#cookies I wouldn’t be surprised if they are the top result on Google for all the keywords they have … 

### “_ We are not responsible for any claims/damages caused by our website. Use it at your own risk_”

![6A50E140-206C-4E94-A272-4C2D1DF519FB](https://user-images.githubusercontent.com/194400/163685295-93598d67-8522-418d-b99d-4a4dd296abdf.jpeg)

I wouldn’t trust a random website in Romania with the health of my child. They don’t have a single link to a medical journal or cite a reference e.g. paediatric textbook. Nobody is taking actual responsibility for the content on the site. It’s like trusting MacDonald’s or Kelloggs with the health of our child. They only care about making money. Not about accuracy/facts. 

Sorry to be harsh. But our *first* instinct when reading something on the Internet should always be: who wrote this? What are their motives? Why/how can I *trust* them? 

Remember Doctors used to recommend smoking:

![DCFD8EA3-0BD1-4839-8D94-9B95BBE00118](https://user-images.githubusercontent.com/194400/163685850-ccf523a0-bad7-4884-9799-29e781439a14.jpeg)

**Comment by Stephanymtr** [comment]
Oh, this is horrible! I 100% agree with you. Feel free to delete my comment 🤦🏻‍♀️ I have to make sure ALL the research results, even if small like this, comes from a trustworthy source. I overlooked all of this. My apologies. I'm hoping comida de bebe will reply my message on IG soon :)

**Comment by nelsonic** [comment]
@Stephanymtr definitely don't worry about linking to something online. 
Apologies if my comment above came across as harsh. 
I just wanted to highlight that I wouldn't trust the site with my child's health.
It's _very_ easy to get sucked into thinking that something on the front page of Google is trustworthy. 🔍 
It's even easier to be lead astray by "influencers" ... 🙄 
No need to `delete` anything it's a good learning point.

A _good_ resource on a topic is Harvard Medical School, 
e.g: https://www.hsph.harvard.edu/nutritionsource/food-features/vinegar/
Another great one is NutritionFacts.org, e.g: https://nutritionfacts.org/?s=vinegar

> In fact the _reason_ I started adding Apple Cidre Vinegar to my Raw Green Soups back in 2016 (despite not liking the taste of it at all!) was because I trust Dr. Greger to be unbiased and provide reliable advice on the topic.
see: https://nutritionfacts.org/2019/01/08/vinegar-is-good-for-you/ 
Or if you prefer video: https://nutritionfacts.org/video/apple-cider-vinegar-help-weight-loss/
I use organic unfiltered Apple Cider Vinegar (with the mother). It costs around €8 /  Litre at Biobrassica. 💸 🙄 


Sadly, neither of these sites have anything on feeding vinegar to ***babies***. 
I tried the following search: 
https://www.google.com/search?q=harvard+medical+school+%22vinegar%22+babies
What does it mean when we cannot find a result from a _reliable_ source?
For me, it means I need to think about it for myself... 💭 

If we think about Vinegar from first principals, the "White Spirit Vinegar" made industrially, the one we use for cleaning @home has a PH of around 2.6 https://en.wikipedia.org/wiki/Vinegar#Spirits if you were to drink it, it wouldn't _kill_ you but it would certainly upset your stomach and cause some damage to your gut microbiome. For reference, CocaCola and Coke Zero have a PH of 2.37 and 2.96 respectively see: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4808596/ 
The active ingredient in Coke is phosphoric acid. Its pH is 2.8. It will dissolve a (metal) nail in about 4 days. https://sciencing.com/science-dissolving-nail-four-days-8743894.html
Typically Balsamic Vinegar pH is usually lower than 3. Meaning it's Similarly acidic to CocaCola. 
See: https://en.wikipedia.org/wiki/Traditional_balsamic_vinegar#Physical_properties

> **Note**: This is not to say that all acidic things are "bad", just that we should use them with caution especially for a child.

I would be weary of any substance that has a significantly low PH as it can destabilise the microbiome.
A good rule of thumb is: if the **substance** is **not _naturally_ occurring** avoid it for children.
Things like refined sugar, artificial sweeteners and other industrially made food additives are out.

This is a good article on the topic: "Common food additives and chemicals harmful to children":
https://www.health.harvard.edu/blog/common-food-additives-and-chemicals-harmful-to-children-2018072414326

Sadly this article doesn't mention vinegar, but that's a "false negative" not a green light to use it.

If you want to dig deeper into Vinegar in Children, please do so on PubMed:
https://pubmed.ncbi.nlm.nih.gov/?term=vinegar+children

E.g. this article published in the journal Nature: https://pubmed.ncbi.nlm.nih.gov/30670726/ cited by 78 other papers that found "Altered gut microbiota and short chain fatty acids in Chinese children with autism spectrum disorder"
![image](https://user-images.githubusercontent.com/194400/163732439-d08e8dbd-b941-42cb-8ac1-4cc7bbe4bdce.png)

i.e: don't mess with your child's microbiome!!
